src/theme/crTheme/linkStyles.js

```diff
@@ -19,6 +19,15 @@ const linkStyles = {
     hoverColor: color('white'),
     hoverBorder: `2px solid ${color('white')}`,
     hoverWeight: '700'
+  },
+  bold: {
+    color: color('black'),
+    border: `2px solid ${color('black')}`,
+    weight: '700',
+    padding: '0 2px 1px',
+    hoverColor: color('black'),
+    hoverBorder: `2px solid ${color('black')}`,
+    hoverWeight: '800'
   }
 };
 
```

Comment:
Can we used spacing variant on padding

Reply:
like a variable? Not sure what you mean by spacing variant

Reply:
Yep, you can check an example on this file `DoubleCopy.style.js`

Reply:
Yeah, it would be nice for us to start using our `spancing` util. ie.`margin-bottom: ${spacing('sm')};` whenever is possible .. margins between elements is a good case for it. Maybe a bit difficult to apply in button padding for instance.. see where it fits best having in mind we need to stay close to current design.

Reply:
ahh ok I see.  I'll have a look to see if it's useful for the link padding. 👍

Reply:
myeah so I think it's not useful for the link padding there. Do you guys want me to introduce a margin-bottom with the spacing variable just in case we need to have multiple buttons on top of each other or something?

Reply:
I don't think so, margins around buttons and link will be defined by its "parent" the component that will be using them and sorting their position etc. Ready to merge! :)